### ISSUE TITLE
[Win 10] 1.50.0 fails to start after upgrade from 1.49.2

### ISSUE THREAD

**Issue by dvorka** [dvorka/mindforger#1163]
Error code:
```
> mindforger
> echo %errorlevel%
```

Hints:
* beast (non-CI) Win installer build 

See #1008

**Comment by sbstn87** [comment]
If I do it in two separate lines, it shows: 0. If I do it in one line, it shows an error window: Unknown option 'disable-web-security'.

**Comment by dvorka** [comment]
@sbstn87 Oh, thanks - this really helps! Seems to be related to #1006 @1005

I'm not Win user and unfortunately it's not reproducible on two installations I have available. I will try to make a fix related to problematic option.

**Comment by dvorka** [comment]
Hints and ideas:

* new Chrome browsers require --user-data-dir
```
[~]$ google-chrome --disable-web-security                                                                                                                                                                           
[0119/212729.578924:ERROR:chrome_main_delegate.cc(625)] Web security may only be disabled if '--user-data-dir' is also specified.
Opening in existing browser session.
```
* other root cause might be that a different browser is linked (which doesn't have this option)

**Comment by dvorka** [comment]
Reproducible after code review:

* if mindforger gets a parameter, then argv/argc is screwd

Repro:
```
mindforger --disable-web-security
mindforger --disable-web-security --user-data-dir

# broken
mindforger --disable-web-security filename
# broken
mindforger --disable-web-security --user-data-dir filename
```

Fix:

* `--disable-web-security` (and possibly `--user-data-dir`) must go first

**Comment by dvorka** [comment]
Fixed by 862fc3948 - closing.
